src/components/Pokemons.js

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { fetchPokemons } from '../redux/actions/pokemonsActions';
+import general from '../style/general.module.css';
+import {
+  fetchSelectedPokemon,
+  getExistingPokemon,
+} from '../redux/actions/clickAction';
+
+const Pokemons = ({
+  select,
+  pokemons,
+  compare,
+  fetchPokemons,
+  getExistingPokemon,
+  fetchSelectedPokemon,
+}) => {
+  useEffect(() => {
+    fetchPokemons(0);
+  }, [fetchPokemons]);
+
+  useEffect(() => {
+    let off = 0;
+    const handleScroll = () => {
+      let scrollHeight = document.documentElement.scrollHeight;
+      let scrollPosition =
+        window.innerHeight + document.documentElement.scrollTop;
+      if ((scrollHeight - scrollPosition) / scrollHeight === 0) {
+        off = off + 20;
+        fetchPokemons(off);
```

Comment:
I believe this can trigger multiple times while the person intention is to load only once, check it out